zanshinsdk/client.py

```diff
@@ -46,6 +48,41 @@ class ScanTargetKind(str, Enum):
     ORACLE = "ORACLE"
 
 
+class ScanTargetSchedule(str, Enum):
+    ONE_HOUR = '1h'
+    SIX_HOURS = '6h'
+    TWELVE_HOURS = '12h'
+    TWENTY_FOUR_HOURS = '24h'
+    SEVEN_DAYS = '7d'
+
+    @classmethod
+    def from_value(cls, schedule: Union[str, ScanTargetSchedule]) -> ScanTargetSchedule:
+        if isinstance(schedule, cls):
+            return schedule
+        elif isinstance(schedule, str):
```

Comment:
I'd accept only the new style, but I understand not wanting to break old clients, if any.

Reply:
Precisely. Don't want to break any existing customer scripts that might rely on the old syntax to work.